### ISSUE TITLE
error data base disk is malformed

### ISSUE THREAD

**Issue by Nanjidesbois** [FarbstoffRSL/RSL-Helper#338]
### Operating system

Windows 11

### Systemlanguage

English (EN)

### Current Game-Version

v3 build 85

### Bug description

la food est introuvable est inexistante
Impossible de sélectionner  les champions  pour l'amélioration!

### Steps to reproduce

j'ai juste cliquer sur set food

### Relevant log output

```shell
(FireDAC)(Phys)(SQlite)
```

**Comment by Dwza** [comment]
Is this still a problem?
And most people dont speak france. So if you would be so kind and my use google translator if you need some help?
you cant expact others to translate for you just to help you :)

**Comment by Nanjidesbois** [comment]
Hello

Still the same problem despite the update, and the installation of version 1 build 30
I do not understand this error messageI wish you a nice end of tournament[Capture d’écran 2022-03-27 133937.png]

Envoyé avec la messagerie sécurisée [ProtonMail](https://protonmail.com/).

------- Original Message -------

**Comment by Dwza** [comment]
have you updated to latest version today?

**Comment by ozmojo67** [comment]
This is still a problem today with V3.1 Release 2 Build 32. Deleted and reinstalled it, and ran the updater.

**Comment by Dwza** [comment]
i still dont know what your actual problem is... what is not working?

**Comment by Nanjidesbois** [comment]
Hello,

First of all thank you for creating this content, it's a nice tool and very useful, the problem I encountered with version 3 build 81, was that I did not have access to my champion base, so I could not use rsl helper to mount my food, the base of the game was invisible in the application, with an error message saying that the image was malformed, I uninstalled and reinstalled, still non-functional, so I tried the version build 85 which still had the same problem. It seems that no one from my clan had this problem, and it was only me. Everywhere else Rsl helper was working.
I'm waiting for the next maj, I know that the work required by the application must be huge.
I wish you a nice day
Thank you for answering me.
See you soon

Translated with www.DeepL.com/Translator (free version)

Envoyé avec la messagerie sécurisée [ProtonMail](https://protonmail.com/).

------- Original Message -------
